src/Pages/Login/Login.scss

```
@@ -0,0 +1,81 @@
+@import "../../Styles/common.scss";
+
+.Login {
+  width: 480px;
+  .loginBox {
+    display: flex;
+    flex-direction: column;
+    .btnClose {
+      height: 44px;
+      display: flex;
+      justify-content: flex-end;
+      button {
+        width: 44px;
+        height: 44px;
+        background-color: white;
+        border: 0;
+        font-size: 25px;
+        font-family: Mulish;
+        outline: none;
+      }
+    }
+    .accountBox {
+      display: flex;
+      flex-direction: row;
+      .accountButton {
+        width: 240px;
+        height: 72px;
+        text-align: center;
+        font-family: Fira-sans;
+        font-size: 28px;
+        font-weight: 900;
+        border: 0;
+        outline: 0;
+        border-bottom: 4px solid #f0f0f0;
+        background-color: #ffffff;
```

css 순서 아시죠? ㅎㅎ 잘 해주셨는데 연관된것 끼리 인접하게 작성하면 더 깔끔하겠네요! 
```suggestion
        width: 240px;
        height: 72px;
        border: 0;
        border-bottom: 4px solid #f0f0f0;
        outline: 0;
        background-color: #ffffff;
        text-align: center;
        font-family: Fira-sans;
        font-size: 28px;
        font-weight: 900;
```

---

수정했습니다 :)